resources/views/livewire/modules/maintenance/fill-filesize-sizevariants.blade.php

```diff
@@ -0,0 +1,6 @@
+<x-maintenance.card :disabled="$this->number_of_missing_size <= 0">
```

Comment:
I guess the `<` is just in case?

Reply:
I think I reused some of the code from gen-missing-sizevariant which was giving weird negative values.
Originally it was `==` but to be on the safe side, `<=` is stronger. :)